index.js

```diff
@@ -184,6 +184,11 @@
         }
       });
 
+      // Add display name for React devtools
+      var componentName = Component.displayName || Component.name || 'Component'
+      var displayName = 'OnClickOustide(' + componentName + ')';
+      wrapComponentWithOnClickOutsideHandling.displayName = displayName;
```

Comment:
can we wrap this in an immediately executing function, so that those two variables don't hang around forever?

Something like:
```
(function bindWrappedComponentName(c, wrapper) {
  var componentName = c.displayName || c.name || 'Component'
  wrapper.displayName = 'OnClickOustide(' + componentName + ')';
}(Component, wrapComponentWithOnClickOutsideHandling));
```